### DISCUSSION TITLE
Not genrerating dist file for in case html file contains lessthan,graterthan or graterthan equal or lessthan equal symbols( <,=,<=,<=)

### DISCUSSION THREAD

**Discussion by Kranthi32** [webpack/webpack#17120] [Q&A]
ERROR in   Error: html-webpack-plugin could not minify the generated output.
  In production mode the html minifcation is enabled by default.
  If you are not generating a valid html output please disable it manually.
  You can do so by adding the following setting to your HtmlWebpackPlugin config:
  |
  |    minify: false
  |
  See https://github.com/jantimon/html-webpack-plugin#options for details.
  For parser dedicated bugs please create an issue here:
  https://danielruf.github.io/html-minifier-terser/
  Parse Error: <,=,<=,<=)

**Comment by alexander-akait** [comment]
Sorry, it is not official plugin, please open an issue in html-webpack-plugin, but maybe you have invalid HTML, so parser can't parse it

**Comment by alexander-akait** [comment]
Sorry, it is not official plugin, please open an issue in html-webpack-plugin, but maybe you have invalid HTML, so parser can't parse it
